lib/src/main/java/com/orange/ods/compose/component/menu/OdsExposedDropdownMenu.kt

```diff
@@ -90,6 +87,14 @@ fun OdsExposedDropdownMenu(
 
 data class OdsExposedDropdownMenuItem(val label: String, val icon: Painter? = null)
 
+/**
+ * This Saver implementation converts OdsExposedDropdownMenuItem object which we don't know how to save to String which we can save.
+ */
+class OdsExposedDropdownMenuItemSaver : Saver<OdsExposedDropdownMenuItem, String> {
```

Comment:
We could annotate `OdsExposedDropdownMenuItem` class with the `Parcelize` annotation instead. That would allow the user to use `rememberSaveable` without passing a custom saver.

In `build.gradle.kts` of `lib`:

```kotlin
plugins {
    ...
    id("kotlin-parcelize")
}
```

And:

```kotlin
@Parcelize
data class OdsExposedDropdownMenuItem(val label: String, val icon: @RawString Painter? = null) : Parcelable
```

The issue with `@Parcelize` is that we need to put `@RawString` to make it compile. However this won't really parcelize the `Painter` (which is already the case with the `OdsExposedDropdownMenuItemSaver`). Thus I would suggest to use an `Int` pointing to a drawable resource instead of a `Painter` (I think these items will always be used with drawable resources anyway).

```kotlin
@Parcelize
data class OdsExposedDropdownMenuItem(val label: String, val icon: Int? = null) : Parcelable
```